core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -278,6 +279,7 @@ object KafkaConfig {
   val ReplicaFetchMaxBytesProp = "replica.fetch.max.bytes"
```

Comment:
I wonder if we should deprecate this in favour of a `replica.fetch.max.partition.bytes`. It seems quite confusing that this one applies to a single partition when every other `replica.fetch.*` setting applies to the request. Maybe something for later.